app/ios/AriesBifold.xcodeproj/project.pbxproj

```diff
@@ -739,7 +703,7 @@
 				OTHER_LDFLAGS = "$(inherited)";
 				REACT_NATIVE_PATH = "${PODS_ROOT}/../../node_modules/react-native";
 				SDKROOT = iphoneos;
-				USE_HERMES = true;
```

Comment:
hermes is the optimized and lightweight version of node that are used for mobile, any reason to turn that off?